src/styles/container.scss

```diff
@@ -0,0 +1,47 @@
+.container {
+  display: grid;
+  width: 100%;
+
+  &__item {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    height: $itemHeight;
+    color: #fff;
+
+    @for $num from 1 to 7 {
+      &--#{$num}::before {
+        content: "#{$num}";
+      }
+    }
+
+    @each $num, $color in $colors {
+      &--#{$num} {
+        background-color: #{$color};
+      }
+    }
+  }
+
+  @media (min-width: 300px) {
+    & {
+      grid-template-columns: 1fr;
+    }
+  }
+
+  @media (min-width: 600px) {
+    & {
+      grid-template-columns: repeat(2, 1fr);
+
+    }
+
+    &__item--1 {
```

Comment:
why do you need this property in media? Your code works even without &__item--1 property.